README.md

```diff
@@ -96,6 +99,67 @@ mongodb-service-0 # Uses some custom k8s service name. Risks being a duplicate e
 If you run the sidecar alongside such a cluster, it may lead to a broken replica set, so make sure to test it well before
 going to production with it (which applies for all software).
 
+#### MongoDB Command
+The following is an example of how you would update the mongo command enabling ssl and using a certificate obtained from a secret and mounted at /data/ssl/mongodb.pem
+
+Command
+```
+        - name: my-mongo
+          image: mongo
+          command:
+            - mongod
+            - "--replSet"
+            - heroku
+            - "--smallfiles"
+            - "--noprealloc"
+            - "--sslMode"
+            - "requireSSL"
+            - "--sslPEMKeyFile"
+            - "/data/ssl/mongodb.pem"
+            - "--sslAllowConnectionsWithoutCertificates"
```

Comment:
I would prefer a safer default without sslAllowConnectionsWithoutCertificates, sslAllowInvalidCertificates and sslAllowInvalidHostnames